### DISCUSSION TITLE
HEIC to JPEGXL conversion failing in Win11

### DISCUSSION THREAD

**Discussion by sim4life** [libvips/libvips#3987] [General]
## Questions, enhancements, tips, etc.

Please use libvips discussions for other topics.

https://github.com/libvips/libvips/discussions

## Bug report

**Describe the bug**
I installed latest libvips v 8.15.2 through scoop package manager on my Win11. When running shell commands to convert from HEIC image to jpegxl image on PowerShell, I get an error `heif: unsupported feature: unsupported codec`.
 
**To Reproduce**
1. Use Image 'IMG_7387.HEIC'
2. Use Win11 23H2, PowerShell, scoop; install libvips v8.15.2 using scoop
3. Run either of these commands on powershell v 7.4.2
`PS D:\> vips jxlsave IMG_7387.HEIC IMG_7387.jxl`
OR
`PS D:\> vips copy IMG_7387.HEIC IMG_7387_ll.jxl[lossless]`
See error output.
Error:
```
(vips.exe:51592): VIPS-WARNING **: 05:43:38.481: error in tile 0 x 0
.
.
(vips.exe:51592): VIPS-WARNING **: 05:43:38.522: error in tile 0 x 14

(vips.exe:51592): VIPS-WARNING **: 05:43:38.526: error in tile 0 x 15
IMG_7387.HEIC: bad seek to 1254156
heif: Unsupported feature: Unsupported codec (4.3000)
```

**Expected behavior**
Expected behaviour is to generate `IMG_7387.jxl` and `IMG_7387_ll.jxl` lossy and lossless images respectively for the two commands because HEIC/HEIF image formats are supported in this latest version of libvips.

**Actual behavior**
Error is produced instead of the out images

**Screenshots**
[N/A]

**Environment**
 - OS: Windows 11 23H2
 - Powershell: 7.4.2
 - scoop: 0.4.2
 - Vips: 8.15.2

**Additional context**
[N/A]

**Comment by lovell** [comment]
It looks like the "scoop" package manager installs the prebuilt "all" variant of libvips from the `build-win64-mxe` repo, which does not include support for the HEVC codec and therefore does not support HEIC images.

https://github.com/ScoopInstaller/Main/blob/master/bucket/libvips.json
